### PR TITLE
update trademarks

### PR DESCRIPTION
**Pull request by orangesurf**

### CODE REVIEW

File: frontend/src/app/shared/components/global-footer/global-footer.component.html
Comment (orangesurf) [inline frontend/src/app/shared/components/global-footer/global-footer.component.html:1]: Untested

File: frontend/src/app/shared/components/global-footer/global-footer.component.html [hunk 1]
```diff
@@ -32,7 +32,7 @@
         </a>
         <p class="explore-tagline-desktop">
           <ng-container i18n="@@7deec1c1520f06170e1f8e8ddfbe4532312f638f">Explore the full Bitcoin ecosystem</ng-container>
-          <ng-template [ngIf]="locale.substr(0, 2) === 'en'"> &trade;</ng-template>
+          <ng-template [ngIf]="locale.substr(0, 2) === 'en'"> &reg;</ng-template>
```
Comment (nymkappa) [inline frontend/src/app/shared/components/global-footer/global-footer.component.html:35]: Need to remove the space

**Review by nymkappa — APPROVED** [review]
Looks fine to me, I've partially tested meta tags and check some page rendering.

Partially tested ACK @ 885331fe4aa78bf3d217892d96e50658fa659fe9

@softsimon feel free to double check

**Review by softsimon — APPROVED** [review]
tested ACK @ [885331fe4]

**Pull request merged**